packages/vscode-extension/src/officeChat/commands/create/officeCreateCommandHandler.ts

```diff
@@ -70,27 +70,34 @@ export default async function officeCreateCommandHandler(
         ),
       ];
       officeChatTelemetryData.chatMessages.push(...describeProjectChatMessages);
-
       await verbatimCopilotInteraction(
         "copilot-gpt-3.5-turbo",
         describeProjectChatMessages,
         response,
         token
       );
+
       if (matchedResult.type === "sample") {
-        const folder = await showOfficeSampleFileTree(matchedResult, response);
+        const sampleInfos = await showOfficeSampleFileTree(matchedResult, response);
+        const folder = (sampleInfos as any)?.["path"];
+        const hostType = (sampleInfos as any)?.["host"].toLowerCase();
         const sampleTitle = localize("teamstoolkit.chatParticipants.create.sample");
+        officeChatTelemetryData.setHostType(hostType);
+        const matchResultInfo = "sample";
         response.button({
           command: CHAT_CREATE_OFFICE_PROJECT_COMMAND_ID,
-          arguments: [folder],
+          arguments: [folder, officeChatTelemetryData.requestId, matchResultInfo],
           title: sampleTitle,
         });
       } else {
-        const tmpFolder = await showOfficeTemplateFileTree(matchedResult.data, response);
+        const tmpHostType = (matchedResult.data as any)?.["addin-host"].toLowerCase();
+        const tmpFolder = await showOfficeTemplateFileTree(matchedResult.data as any, response);
```

Comment:
Same as above, use the defined object struct.

Reply:
the type of matchedResult.data is unknown so that I think it should be convert to any